sdk/python/feast/feature_store.py

```diff
@@ -599,7 +612,17 @@ def _entity_row_to_field_values(
     return result
 
 
-def _group_refs(
+def _validate_feature_refs(feature_refs: List[str], full_feature_names: bool = False):
```

Comment:
Should we validate that the same feature name does not occur in the same feature view more than once?

Reply:
Yes, that seems necessary. I just pushed the change, see comment above for example!